src/main/java/vg/civcraft/mc/civchat2/CivChat2Manager.java

```diff
@@ -73,6 +77,24 @@ public CivChat2Manager(CivChat2 pluginInstance) {
 		scoreboardHUD = new ScoreboardHUD();
 	}
 
+	/**
+	 * Used to set the slowmodeTime of global chat.
+	 * @param amount of seconds the slowmode should be set to.
+	 */
+	public static void setSlowmodeTime(Integer time) {
+		if(time < 0) return;
+		slowmodeTime = time;
+	}
+
+	/**
+	 * Used to fetch the slowmodeTime of global chat.
+	 * @return Returns amount of seconds the slowmode is set to.
+	 */
+	public static int getSlowmodeTime() {
```

Comment:
Shouldn't be static. I can see that that makes accessing it easier, but depending on instanciation of manager classes leads to better encapsulation and prevents bugs.